ResearchKit/Common/ORKPasscodeViewController.h

```diff
@@ -66,6 +66,22 @@ ORK_AVAILABLE_DECL
  */
 - (void)passcodeViewControllerDidCancel:(UIViewController *)viewController;
 
+/*
```

Comment:
Consistency. The other delegate methods include it and it's the standard for Apple delegate protocols.